Core/Configuration/JsonConfiguration.cs

```diff
@@ -370,7 +370,7 @@ private void LoadConfig()
                     // Ensure the directory exists
                     new FileInfo(configFile).Directory.Create();
 
-#if !NETSTANDARD
+#if !NETCOREAPP
                     // If we are not running on .NET Standard, try to migrate from the real registry
```

Comment:
Should this comment about .NET Standard be updated now that the condition is no longer `!NETSTANDARD`?